serverless.yml

```diff
@@ -46,103 +46,155 @@ functions:
   GetPrepayments:
     handler: API/Functions/Payments.get
     name: PrepaymentsRepo-get-${opt:stage}
-    environment:
-      BODY_REQUEST: ${file(./config/${opt:stage}.json):BODY_REQUEST}
-      PHONE_NUMBER: ${file(./config/${opt:stage}.json):PHONE_NUMBER}
     events:
       - http:
           path: /prepayments
           method: GET
+          integration: lambda
           cors:
             - enabled: true
             - origin: "*"
             - headers: "Access-Control-Allow-Origin"
+          response:
+            headers:
+              Access-Control-Allow-Origin: "'*'"
+          authorizer:
+            name: authorizer
+            arn: ${file(./config/${opt:stage}.json):USER_POOL_ARN}
```

Comment:
I think we can deploy the ARN using the Cloud formation sintax, another issue 🤘🏼